frame/contracts/src/wasm/runtime.rs

```diff
@@ -2466,6 +2471,43 @@ pub mod env {
 		}
 	}
 
+	/// Verify the sr25519 signature given the message and public key
+	///
+	/// # Parameters
+	///
+	/// - `signature_ptr`: the pointer into the linear memory where the signature is placed. Should
+	///   be decodable as a 64 bytes. Traps otherwise.
+	/// - `message_ptr`: the pointer into the linear memory where the message is placed.
+	/// - `message_len`: the length of the message payload. Should be decodable as a Scale encoded
+	///   Vec<u8>. Traps otherwise.
+	/// - `pub_key_ptr`: the pointer into the linear memory where the public key is placed. Should
+	///   be decodable as a 32 bytes. Traps otherwise.
+	/// # Errors
+	///
+	/// - `ReturnCode::Sr25519VerifyFailed
+	#[prefixed_alias]
```

Comment:
```suggestion
	#[unstable]
	#[prefixed_alias]
```
new functions not covered with e2e tests should be marked _unstable_